imap_processing/codice/utils.py

```diff
@@ -5,7 +5,46 @@
 other CoDICE processing modules.
 """
 
+import json
 from enum import IntEnum
+from pathlib import Path
+
+import numpy as np
+
+from imap_processing.spice.time import met_to_ttj2000ns
+
+
+class ViewTabInfo:
```

Comment:
Looks like a dataclass or namedtuple?
```suggestion
@dataclass
class ViewTabInfo:
        apid: int
        view_id: int
        sensor: int
        three_d_collapsed: int
        collapse_table: int
```